src/pymordemos/elliptic2.py

```diff
@@ -6,18 +6,21 @@
 """Simple demonstration of solving the Poisson equation in 2D using pyMOR's builtin discretizations.
 
 Usage:
-    elliptic2.py [--fv] PROBLEM-NUMBER N
+    elliptic2.py [--fv] PROBLEM-NUMBER N PRODUCT-COUNT
 
 Arguments:
-    PROBLEM-NUMBER    {0,1}, selects the problem to solve
-    N                 Triangle count per direction
+    PROBLEM-NUMBER  {0,1}, selects the problem to solve
+    N               Triangle count per direction
+    PRODUCT-COUNT   0: equip the model with standard products
+                    1: add an energy product for a fixed parameter
```

Comment:
Are you sure that this isn't causing the problem with the indentation?

Reply:
you mean the 0: and 1: enumeration? This is done the same way in https://github.com/pymor/pymor/blob/master/src/pymordemos/elliptic.py .
Anyways, I might have fixed the indention error.

Reply:
Ah, I see, you're right.